### ISSUE TITLE
Get notified when saving metadata if required fields are not filled in

### ISSUE THREAD

**Issue by cmharlow** [LD4P/sinopia_editor#303]
I5: Get notified when saving metadata if required fields are not filled in

(is this really, just don't allow to Post / Save until all required fields are filled in?)

**Comment by michelleif** [comment]
duplicate of LD4P/sinopia_editor#303

**Comment by jgreben** [comment]
I think this is blocked by https://github.com/LD4P/sinopia_editor/issues/313 (?)

**Comment by jgreben** [comment]
Depends on #134

**Comment by justinlittman** [comment]
@michelleif Are there requirements / comps on how validation errors should be displays? (Inline, dialog?) Thanks!

**Comment by michelleif** [comment]
no requirements. ideally, we would tell the user which fields still need filling in, maybe by highlighting those fields.
but given schedule, we will be happy with something as simple as a message "please fill in all required fields--required fields are marked with an asterisk (*)"
